robot/EDA/resources/EDA.robot

```diff
@@ -1,9 +1,46 @@
 *** Settings ***
 
-Resource       cumulusci/robotframework/Salesforce.robot
-Library        EDA.py
+Resource        cumulusci/robotframework/Salesforce.robot
+Library         robot/EDA/resources/EDA.py
+Library         DateTime
+
+*** Variables ***
+${PRINT_PACKAGE}        true
 
 *** Keywords ***
+Open Test Browser And Print Package Details
+    [Documentation]         Opens the test browser and runs a keyword that prints package details from setup
+    Open Test Browser
+    Run Keyword If          '${PRINT_PACKAGE.lower()}' == 'true'     Capture SAL and EDA package details
+
+
+Capture SAL and EDA package details
+    [Documentation]         Captures the package details from SAL setup page and prints it to console
+    Set Global Variable                 ${PRINT_PACKAGE}            false
+    Wait Until Loading Is Complete
+    Go To Setup Home
+    Wait For New Window                 Home | Salesforce
+    Select Window                       Home | Salesforce
+    Wait Until Loading Is Complete
+    Populate Placeholder                Quick Find          Installed Packages
+    Wait Until Loading Is Complete
+#    Select Frame With Value             Installed Packages ~ Salesforce -
+#    Print Package Details
+#    Unselect Frame
+
+Capture Screenshot and Close Browser If Tests Passed
```

Comment:
Is this keyword used anywhere? I don't see it being used anywhere. If it's not being used, it should be deleted.

Reply:
not sure who wrote this code - it was some other work not related to this work item.  This will be cleaned up in the EDA rewrite.

Reply:
If you're adding code that isn't being used anywhere, you're just adding technical debt. There's no reason to save the removal of this code for a rewrite. If you aren't using it, why add it in the first place?

Reply:
Removed technical debt.